admin/run

```diff
@@ -50,13 +50,26 @@ command_full_path=$(which "$1")
 command_verbatim=$1
 shift
 
+if ! id -u 'musicbrainz' &>/dev/null
+then
+  echo >&2 "$SCRIPT_NAME: user 'musicbrainz' doesn't exist"
+  echo >&2 "Make sure to run in a production container."
+  exit 67 # EX_NOUSER
+fi
+
+if [[ "$(id -u -n)" != 'musicbrainz' ]]
+then
+  sudo --preserve-env --set-home -u musicbrainz -- ${BASH_SOURCE[0]}
```

Comment:
Thanks, I didn't think of that. :)

Does that need `exec`, or should the current script exit after this line?

Reply:
I also forgot to path the arguments. All fixed in https://github.com/metabrainz/musicbrainz-server/pull/2968/commits/4c482cbbcc7a1259c46dd66c84d9162a55a3dfc7